package.json

```diff
@@ -11,12 +11,15 @@
     "commitlint": "commitlint -e",
     "format": "prettier --write '.{/**,}/*.{md,js,ts}' 'package.json'",
     "lint-commit": "commitlint -e $GIT_PARAMS",
-    "pretest": "npm run format",
+    "pretest":
+      "PROXY_HOST_PORT='localhost:4241' npm start& echo $! > .integration-test-server-pid && sleep 3",
```

Comment:
what is this for?

Reply:
Please see my previous comment: https://github.com/mesosphere/mockserver/pull/15#discussion_r182059050